src/game/hud.cpp

```diff
@@ -698,6 +700,11 @@ namespace hud
         result(list.getbuf());
     });
 
+    bool show_system_time()
```

Comment:
This getter is kind of pointless given the fact the variable is accessible globally already.